providers/fips/fipsprov.c

```diff
@@ -697,15 +697,8 @@ int OSSL_provider_init_int(const OSSL_CORE_HANDLE *handle,
 
     /*  Create a context. */
     if ((*provctx = ossl_prov_ctx_new()) == NULL
-        || (libctx = OSSL_LIB_CTX_new()) == NULL) {
-        /*
```

Comment:
You sure about this change?

Reply:
Yes.  `libctx` is NULL before the if.  So it must still be NULL on an error.
Dead code.